general/odoo_basics/add_user.rst

```diff
@@ -1,46 +1,141 @@
-=================
-How to add a user
-=================
+==================================
+Add Users and Manage Access Rights
+==================================
 
-Odoo provides you with the option to add additional users at any given
-point.
+Odoo defines a *user* as someone who has access to a database to perform daily tasks. You can add as
+many users as you need, and in order to restrict the type of information each user can access, rules
+can be applied. Users and access rights can be added and changed at any point.
+
+.. _general/odoo_basics/add_individual_user:
 
 Add individual users
 ====================
 
-From the Settings module, go to the submenu :menuselection:`Users --> Users` and click on
-**CREATE.** First add the name of your new user and the professional
-email address - the one he will use to log into Odoo instance - and a
-picture.
+Go to :menuselection:`Settings --> Manage Users` and click on *Create*.
+
+.. image:: media/manage_users.png
+   :align: center
+   :height: 280
+   :alt: View of the settings page emphasizing the manage users field in Odoo
 
-.. image:: media/add_user01.png
-    :align: center
+| Fill in the form with the needed information. Under the tab
+  :ref:`Access Rights <general/odoo_basics/add_user/access_rights>` choose the group within
+  each application the user can have access.
+| The list of applications shown is based on the applications you have installed on the database.
 
-Under Access Rights, you can choose which applications your user can
-access and use. Different levels of rights are available depending on
-the app.
+.. image:: media/new_user.png
+   :align: center
+   :alt: View of a user’s form emphasizing the access rights tab in Odoo
 
-When you’re done editing the page and have clicked on **SAVE**, an
-invitation email will automatically be sent to the user. The user must
-click on it to accept the invitation to your instance and create a
-log-in.
+When you are done editing the page and have *Saved* it, an invitation email is automatically sent to
+the user. The user must click on it to accept the invitation to your instance and create a login.
```

Comment:
```suggestion
When you are done editing the page and have *Saved* it, an invitation email is automatically sent to
the user. The user must click on it to accept the invitation and create a login.
```

Reply:
From my understanding and research, the term is correct.